hack/release-local.sh

```diff
@@ -19,10 +20,10 @@ REV=$(git rev-parse --short HEAD)
 TAG="${TAG:-$REV}"
 
 if [[ -z "${DOCKER+1}" ]] && command -v buildah >& /dev/null; then
-  buildah bud -t $REPO:$TAG .
+  buildah bud -t $REPO:$TAG -f "${DOCKERFILE}" .
   buildah push $REPO:$TAG docker://$REPO:$TAG
 else
-  docker build -t $REPO:$TAG .
+  docker build -t $REPO:$TAG -f "${DOCKERFILE}" .
```

Comment:
I think we want to prefer podman over docker when possible. Since you're changing this, would this be a good opportunity to add an `elif` block to use podman if it's present?

Reply:
> I think we want to prefer podman over docker when possible. Since you're changing this, would this be a good opportunity to add an `elif` block to use podman if it's present?

Or add: 

```
: "${PODMAN:=podman}"
```

and you can override PODMAN on invocation. By default we'd just use podman though.
You'd obviously have to replace all the plain invocations of `podman` with `$PODMAN`.

Reply:
> I thought of making a smooth change without any impact on the potential users of the script

That's a fair point. If don't want to perturb what's already there we could invert my change to DOCKER=docker which would allow the status quo but also make it easy if you only have podman installed.

Reply:
This wasn't clear from my original comment (sorry :pensive:), but I was thinking that we would prefer podman over docker, but leave buildah as the default. I know podman is an acceptable substitute to docker since that's what I use locally ("docker" on my machine is a symlink to podman), but I don't know if there's anything we'd be losing by preferring it over buildah.

Reply:
Done. The order is:`buildah`, `podman`, `docker`.